app/controllers/events_controller.rb

```diff
@@ -4,6 +4,15 @@ class EventsController < InheritedResources::Base
   skip_load_and_authorize_resource only: [:index, :show]
   before_filter :set_types
 
+  def participants
+    @breadcrumbs = [
+      { label: 'events', url: events_path },
+      { label: @event.name, url: event_path(@event) },
+      { label: 'participants' }
+    ]
+    @users = @event.users.travel
```

Comment:
I don't think that this is a good idea as it would move the ability logic out of the ability file ...